glean-core/ios/Glean/Debug/GleanDebugTools.swift

```diff
@@ -100,10 +100,10 @@ class GleanDebugUtility {
             }
 
             switch param.name {
-            case "tagPings":
+            case "debugViewTag":
```

Comment:
I assume we don't need to keep backward-compatibility here because ... well, there's just not enough people out there that knew the old way?

Reply:
And here I spend all this time teaching the iOS folks about this... :D  

I'll do a PSA about this to the iOS folks that know about it just to be friendly.

Reply:
Wait :) There will be a PSA about this aimed at all